.github/workflows/linux.yml

```diff
@@ -9,10 +9,32 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.4', '2.5', '2.6', '2.7' ]
+        ruby: [ '2.5', '2.6', '2.7', '3.0' ]
         postgres: [ '9', '10', '11', '12', '13' ]
         os:
           - ubuntu-latest
+        experimental: [false]
```

Comment:
Could you add `continue-on-error: ${{ matrix.experimental }}`?
Otherwise it doesn't take effect.

Reply:
Fixed in https://github.com/fluent/fluent-plugin-sql/pull/101/commits/7496d87dbce69a8cc644bd767741107ee751a292.